### DISCUSSION TITLE
Color Temperature selection should be a slider, not a wheel

### DISCUSSION THREAD

**Discussion by W-Floyd** [home-assistant/frontend#16814] [Other feature requests] [answered]
Unless I'm thoroughly mistaken, color temperature is a single dimensional variable. Why is it represented by a wheel, instead of a slider? It makes it difficult to exactly set the maximum/minimum color temperature. It also appears confusing as the same y value can be selected across the x axis - what difference does this x position make? Changing to the color wheel and back resets it to the center of the wheel anyway, so why bother?

https://github.com/home-assistant/frontend/assets/11786132/8ca7b34f-b6bb-4339-9a41-d30c172bace1

**Comment by flabbamann** [comment]
Yes, I think this is confusing and less intuitive as the old slider.

**Comment by piitaya** [comment]
Hello 👋
The change has been done to have consistency between the color wheel and the temperature color selector, to have seamless switch because color picker and temperature picker. The new control better reflect the color temperature color (the gradient change with min and max temperature).

Other pickers (like iro.js) always center the cursor but the UX feels strange.

https://github.com/home-assistant/frontend/assets/5878303/bfcc2312-6cf7-417b-a7ab-71ab03e4ebd2

I do understand your concern. If the main issue if about settings to min and max temperature, it can be fixed by adding a safe zone for the min and max value.

Example: 

https://github.com/home-assistant/frontend/assets/5878303/2328c81b-10a5-4665-a25f-220ac73d6e12

**Reply by W-Floyd** [reply]
I think the safe zone would do a lot to make it as functional as a slider, even if a real slider is my preference.

I have no clue if we'd be able to do this, but could we have a horizontal bar instead of a dot, so like a circular slider? I can mock something up for what I'm visualizing, just not sure how it's built out.

This way, dragging diagonally won't feel so unusual (I do this regularly on sliders), while still retaining the shared circular shape.

**Comment by piitaya** [comment, marked as answer]
Reverted back the control to a slider based on users feedbacks : https://github.com/home-assistant/frontend/pull/17771
